src/logic/Application.swift

```diff
@@ -125,7 +125,7 @@ class Application: NSObject {
     private func addWindows(_ axWindows: [(AXUIElement, CGWindowID, String?, Bool, Bool, CGPoint?, CGSize?)]) -> [Window] {
         let windows: [Window] = axWindows.compactMap { (axUiElement, wid, axTitle, isFullscreen, isMinimized, position, size) in
             if (Windows.list.firstIndex { $0.isEqualRobust(axUiElement, wid) }) == nil {
-                let window = Window(axUiElement, self, wid, axTitle, isFullscreen, isMinimized, position, size)
+                let window = Window(axUiElement, self, wid, axTitle, isFullscreen, isMinimized, position, size, Int(NSDate().timeIntervalSince1970))
```

Comment:
I think it would be easier to use a simple `Int` index here, like is done for `lastFocusOrder`. You can simply increment from the highest index when a new window is spawned. When windows are deleted, you don't even need to do anything, the order will work still. Regarding a potential overflow, it would deal with `9223372036854775807` (`Int.max`) windows being opened, before overflowing, so I think it's fine.

This would make the code easier, faster, and simpler.